src/pe.rs

```diff
@@ -2001,11 +2001,13 @@ pub struct ImageResourceDirectory {
     pub minor_version: U16<LE>,
     pub number_of_named_entries: U16<LE>,
     pub number_of_id_entries: U16<LE>,
-    // DirectoryEntries[ImageResourceDirectoryEntry];
 }
 
 pub const IMAGE_RESOURCE_NAME_IS_STRING: u32 = 0x8000_0000;
 pub const IMAGE_RESOURCE_DATA_IS_DIRECTORY: u32 = 0x8000_0000;
+pub const IMAGE_RESOURCE_NAME_OFFSET_MASK: u32 = 0x7FFF_FFFF;
+pub const IMAGE_RESOURCE_NAME_ID_MASK: u32 = 0x0000_FFFF;
+pub const IMAGE_RESOURCE_DATA_OFFSET_MASK: u32 = 0x7FFF_FFFF;
```

Comment:
Try to avoid adding definitions in this file unless they are copied from authoritative sources. They can be placed in the file doing the parsing instead.

Reply:
Removed